source/whats-new.txt

```diff
@@ -58,6 +59,69 @@ Learn what's new in:
 * :ref:`Version 3.7 <version-3.7>`
 * :ref:`Version 3.6 <version-3.6>`
 
+.. _version-6.8:
+
+What's New in 6.8
+-----------------
+
+The {+driver-short+} v6.7 release includes the following features:
+
+- Fixes a bug where a local KMS provider accepted a ``BSON Binary`` instance at
+  runtime, but the TypeScript compiler allowed only values of type ``Buffer`` and
+  ``string``.
+
+- The ``ReadConcernMajorityNotAvailableYet`` error is now a retryable read error.
+
+- KMS providers can now be associated with a name, and multiple keys can be provided
+  for each KMS provider. This feature requires ``mongodb-client-encryption`` v6.0.1 or later.
+  You can't use named KMS providers if your application uses the automatic
+  KMS provider refresh capability.
+
+  The following code example shows how to configure a ``ClientEncryption`` object with
+  multiple AWS keys:
+
+.. code-block:: javascript
+   :copyable: true
+   
+   const clientEncryption = new ClientEncryption(keyVaultClient, {
+     'aws:key1': {
+       accessKeyId: ...,
+       secretAccessKey: ...
+     },
+     'aws:key2': {
+       accessKeyId: ...,
+       secretAccessKey: ...
+     },
+  
+   clientEncryption.createDataKey('aws:key-1', { ... });
+
+- When you create a KMIP data key, you can now specify the ``delegated`` option. If this
+  option is set to ``true``, the KMIP provider performs encryption and decryption of
+  the data key locally, ensuring that the encryption key never leaves the KMIP server.
+  This feature requires ``mongodb-client-encryption`` v6.0.1 or later.
+  
+  The following code example shows how to specify this option:
+
+.. code-block:: javascript
+   :copyable: true
+
+   clientEncryption.createDataKey('kmip', { masterKey: { delegated: true } } );
+
+- The driver now decodes BSON responses as the cursor iterates over them,
+  rather than decoding the entire BSON response when it is received.
+
+- The Github release for the ``mongodb`` package now contains a detached signature file for
+  the NPM package (named ``mongodb-X.Y.Z.tgz.sig``). This change applies to every major
```

Comment:
I don't think these parenthesis are necessary
```suggestion
  the NPM package ``mongodb-X.Y.Z.tgz.sig``. This change applies to every major
```

Reply:
i think the file name refers to the signature file, not the NPM package. reworded anyway